scripts/bump_version.py

```
@@ -19,20 +20,224 @@
+    >>> entry = tomlkit.table()
+    >>> entry["version"] = tomlkit.string("^0.1")
+    >>> _update_dict_dependency(entry, "1.2.3")
+    >>> entry["version"].value
+    '^1.2.3'
+    """
+    prefix = _extract_version_prefix(entry)
+    entry["version"] = prefix + version
+
+
+def _update_string_dependency(
+    deps: dict, dependency: str, entry, version: str
```

Ruff (0.12.2)

Missing type annotation for function argument `entry`

(ANN001)